tasks/libs/github_actions_tools.py

```diff
@@ -191,7 +191,10 @@ def print_failed_jobs_logs(run):
 
     for failed_step, log_file in failed_steps.items():
         print(color_message(f"Step: {failed_step} failed:", "red"))
-        print("\n".join(parse_log_file(log_file)))
+        try:
+            print("\n".join(parse_log_file(log_file)))
+        except FileNotFoundError:
+            print(f'Failed to parse {log_file}, file does not exist')
```

Comment:
It would be helpful for the error message to give the link to the GitHub Action, so the user can check it directly - or add that link in the message below (which already mentions that for complete logs, you should check the workflow).